scripts/merge-and-build.sh

```diff
@@ -175,10 +175,9 @@ if [ "${OSE_VERSION}" != "3.2" ] ; then
   VC_COMMIT="$(GIT_REF=enterprise-${OSE_VERSION} hack/vendor-console.sh 2>/dev/null | grep "Vendoring origin-web-console" | awk '{print $4}')"
   git add pkg/assets/bindata.go
   git add pkg/assets/java/bindata.go
-  set +e # Temporarily turn off errexit. THis is failing sometimes. Check with Troy if it is expected.
-  # This fails only when there is nothing new to commit, which is normal.
-  git commit -m "[DROP] bump origin-web-console ${VC_COMMIT}"
-  set -e
+  if [ "$(git status --porcelain)" ]; then
```

Comment:
I would prefer this to be more explicit:

```sh
if [[ -n "$( git status --porcelain )" ]]; then
```

Also, the basic "does `git status` have output" check doesn't make a distinction between staged and unstaged changes, which does matter for `git commit`. If this is in the rebase script, though, again, won't we always have changes since we're blowing away old changes and OCP never tracks Origin Web Console master?